packages/generator/src/generator.ts

```diff
@@ -55,33 +54,47 @@ const logger = createLogger({
   messageContext: 'generator'
 });
 
-export async function generate(
-  options: GeneratorOptions
-): Promise<void | EmitResult[]> {
+const threadsForCompiling = 16;
```

Comment:
Moving it to the cli args?

Reply:
I also head the idea, but was to lazy because this would also mean an adjustment in the cli. But I will add it and make a follow up ticket.